test/e2e/framework/volume/fixtures.go

```diff
@@ -619,6 +615,13 @@ func generateWriteFileCmd(content, fullPath string) []string {
 
 // CheckVolumeModeOfPath check mode of volume
 func CheckVolumeModeOfPath(f *framework.Framework, pod *v1.Pod, volMode v1.PersistentVolumeMode, path string) {
+	// in windows a symlink is created instead of mounting a volume
+	// we just check if the symlink exists
+	if framework.NodeOSDistroIs("windows") {
+		VerifyExecInPodSucceed(f, pod, fmt.Sprintf("ls %s", path))
```

Comment:
isn't test.exe available on Windows?

Reply:
thank you for the review, it's indeed available in windows and it works for the GCE PD CSI test as it is, I think I added these lines before making sure that the pod was starting correctly but didn't remove them because I thought they were part of the fix, testing in windows is sort of trial and error

For reference these commands work with a FS volumeMode in windows (I'm not sure if `test -b` works with a block volumeMode in windows because as far as I know in windows we just create a symlink)

```
C:/ $ test -d /mnt/volume1
C:/ $ echo $?
0
C:/ $ test -b /mnt/volume1
C:/ $ echo $?
1
```